lib/platform/bitbucket/index.spec.ts

```diff
@@ -531,10 +531,18 @@ describe('platform/bitbucket', () => {
           '/2.0/repositories/some/empty/issues?q=title%3D%22title%22%20AND%20(state%20%3D%20%22new%22%20OR%20state%20%3D%20%22open%22)%20AND%20reporter.username%3D%22abc%22'
         )
         .reply(200, { values: [] })
+        .get(
+          '/2.0/repositories/some/empty/issues?q=title%3D%22old-title%22%20AND%20(state%20%3D%20%22new%22%20OR%20state%20%3D%20%22open%22)%20AND%20reporter.username%3D%22abc%22'
```

Comment:
We can maybe remove the title migration in the next major release so that Bitbucket Cloud doesn't have this extra query each time.